content/2017.12.kinto-at-mozilla.rst

```diff
@@ -0,0 +1,148 @@
+Kinto at Mozilla
+################
+
+:lang: en
+:date: 2017-12-20
+:url: kinto-at-mozilla
+:summary: A summary of how Kinto is used at Mozilla
+
+
+During the last company «all hands», we've heard many mentions of *Kinto*, mostly from people that we hadn't met before or that wasn't aware we were the authors. Part of this fame may probably be due to the success of Firefox Notes, and the recent addition of the long awaited synchronization features.
+
+It's amazing how we've accomplished with this project in very heterogenous contexts, and sadly we haven't really taken the time to celebrate. This post intends to present the different use-cases we are aware of at Mozilla, with some architecture and metrics details.
+
+
+Firefox remote settings
+=======================
+
+The historical one!
```

Comment:
If I understood correctly here, I would probably write `original` instead of `historical`.

(At first I didn't know exactly how to understand `one`. But reading further I understand the joke and like it.)